### PR TITLE
[3460][ADD] base_tier_validation, sale_tier_validation, purchase_tier_validation

### PR DESCRIPTION
**Pull request by rinaldifirdaus**
[3460](https://www.quartile.co/web?db=quartile&token=dMRr78CMkEX6gyy9nTiv#id=3460&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by yostashiro** [comment]
@rinaldifirdaus A rather minor point - we should split the PR per module this time, as we are not sure both sale_tier_validation and purchase_tier_validation will be used.

File: base_tier_validation/i18n/ja.po [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+#, python-format
+msgid "A review was accepted. (%s)"
+msgstr "承認依頼は承認されました。"
+
+#. module: base_tier_validation
+#. odoo-python
+#: code:addons/base_tier_validation/models/tier_validation.py:0
+#, python-format
+msgid "A review was rejected by %(user)s. (%(comment)s)"
+msgstr "承認依頼は拒否されました。"
```
Comment (kanda999) [inline base_tier_validation/i18n/ja.po]: Do %(user)s. and (%(comment)s) need to be added to the translation?

File: base_tier_validation/i18n/ja.po [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+
+#. module: base_tier_validation
+#. odoo-python
+#: code:addons/base_tier_validation/models/tier_validation.py:0
+#, python-format
+msgid "A review was rejected by %s."
+msgstr "承認依頼は拒否されました。"
```
Comment (kanda999) [inline base_tier_validation/i18n/ja.po]: Do ％s need to be to the translation?

File: base_tier_validation/i18n/ja.po (hunk 1 above)
Comment (rinaldifirdaus) [reply base_tier_validation/i18n/ja.po]: ah yes, i missed that part. i will add it.

File: base_tier_validation/i18n/ja.po (hunk 2 above)
Comment (rinaldifirdaus) [reply base_tier_validation/i18n/ja.po]: yes i will add the missing string.

**Review by kanda999 — APPROVED** [review]

**Pull request merged**